faster_sam/middleware.py

```diff
@@ -53,3 +58,60 @@ async def dispatch(self, request: Request, call_next: RequestResponseEndpoint) -
 
         request.scope["path"] = request.scope["path"].replace(self.path, "", 1)
         return await call_next(request)
+
+
+class LambdaAuthorizer(BaseHTTPMiddleware):
+    """
+    Invoke lambda function in aws to authorize apis
+
```

Comment:
Talvez adicionar um `e.g` para manter o padrão?

Veja um exemplo:

    e.g

    This example apply the middleware to transform the request path from "/foo/bar" to "/bar"
    by removing a specified part ("/foo").